tlspuffin/src/tls/rustls/tls13/key_schedule.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code)]
```

Comment:
This is not really a part of the code that I explored a lot so it's hard to form an opinion on what might be useful in the near future. Also, that there were/are several PR opened which might or might not need this code and I didn't want to check them.

If you are positive that unused parts in the rustls-imported code can be removed, I'd be happy to add a commit for that. I'm always strangely enthusiastic about removing code 😄

Reply:
Haha, yeah best PRs have a negative added lines count :D

I mean, the full TLS key schedule is currently implemented afaik. And this code has been lying there and emitting warnings since forever. The git commit of rustls is clearly documented in the mod file as far as I can remember. So if you have spare cycles I'm happy to remove all unused stuff.

At this point we will no longer benefit from changes in rustls. `rustls` has diverged too much by now.

Reply:
But maybe let's do that in another PR?

Reply:
I agree it might be better to keep that for another PR. I think keeping the scope of this one small is a must, as I still have a few more PRs planned after this one is merged 😅